model/feature_flags.go

```diff
@@ -97,6 +99,7 @@ func (f *FeatureFlags) SetDefaults() {
 	f.AdvancedTextEditor = true
 	f.CallsEnabled = true
 	f.BoardsProduct = false
+	f.ServerUpgradeButtonText = "A"
```

Comment:
nit: can we provide a more meaningful value for this ? Maybe `Upgrade` and `ViewPlans`. It's not obvious here what the default values is